src/pages/studyView/table/FixedHeaderTable.tsx

```diff
@@ -47,6 +50,24 @@ const RVSDTtoStrType = {
     ['asc' as SortDirection]: RVSortDirection.ASC
 };
 
+export class FixedHeaderTableDataStore extends SimpleGetterLazyMobXTableApplicationDataStore<any> {
+
+    constructor(getData: () => any[], fixedTopRowsData: any[]) {
+        super(getData);
+        this.fixedTopRowsData = fixedTopRowsData;
+    }
+
+    @observable private fixedTopRowsData: any[];
```

Comment:
minor but seems like fixedTopRows is actually immutable from perspective of this store.  it's never changed during life of store instance.  if so, it doesn't need to be observable.